README.md

```diff
@@ -3,18 +3,13 @@
 
 An elemental block to enable more control over the responsiveness of tables.
 
-## Require via composer.json
-
-```json
-"repositories": [
-    {
-        "type": "vcs",
-        "url": "git@gitea:Factory/silverstripe-factory-elemental-tables.git"
-    }
-],
-"require": {
-    "signify-nz/silverstripe-factory-elemental-tables": "dev-master"
-}
+## Installation
+
```

Comment:
This module isn't publicly available (yet), so these instructions aren't right. I wanted the URL in the `repositories` section to be updated (to the GitHub one), and the version to be changed from `dev-master` to `^1`.